connector/saml/saml.go

```diff
@@ -466,21 +466,25 @@ func (p *provider) validateConditions(assertion *assertion) error {
 		}
 	}
 	// Validates audience
+	audienceValue := p.entityIssuer
```

Comment:
I think it'll be ok as long as we make it clear in the documentation. Let me add it.